jaeger-core/src/main/java/com/uber/jaeger/reporters/protocols/ThriftSpanConverter.java

```diff
@@ -74,7 +74,7 @@
     List<LogData> logs = span.getLogs();
     if (logs != null) {
       for (LogData logData : logs) {
-        annotations.add(new Annotation(logData.getTime(), logData.getMessage()));
+        annotations.add(new Annotation(logData.getTime(), logData.getMessage()).setHost(host));
```

Comment:
I do not believe this is necessary. Can you demonstrate the instrumentation code that results in non-searchable annotations? AFAIK the (old Scala) zipkin backend will extract service name from *any* annotations found on the span, and use that service name for indexing all other annotations. We already provide service name in the core annotations, _and_ in the `lc` (local component) annotation for non-rpc spans. Including service name in *all* annotations that come from the span logging API is only going to bloat the span size.

cc @adriancole